### DISCUSSION TITLE
Info on Version 9?

### DISCUSSION THREAD

**Discussion by matthias-ccri** [openlayers/openlayers#15520] [General]
Hi all!

Is there any way I can learn about the upcoming version 9, and what breaking changes it entails?

I saw that v9 prerelease tags are being published to [npm](https://www.npmjs.com/package/ol?activeTab=versions)

**Comment by ahocevar** [comment]
Thanks for your interest in version 9! You can `npm install ol@dev` to try it, and you can browse the [upgrade notes](https://github.com/openlayers/openlayers/blob/main/changelog/upgrade-notes.md) to learn about the breaking changes.

**Reply by matthias-ccri** [reply]
Perfect, thanks! I didn't notice the `changelog` directory.
